### ISSUE TITLE
Display project ID and module version during cloud debug session

### ISSUE THREAD

**Issue by patflynn** [GoogleCloudPlatform/gcloud-intellij#234]
The user should be able to identify which project and version they are 'connected' to during a debug session.

**Comment by elharo** [comment]
Good idea. Where should we display this information? Maybe we can add a JLabel to the title bar of the debugger tab since we've removed almost everything else that used ton live there.

**Comment by patflynn** [comment]
SGTM.

**Comment by elharo** [comment]
decided to limit IDEA to running a single CDB session at a time, which makes this low priority.

If we do pick it up, it would be nice to figure out how to add this to the tab title for the run/debug configuration tab.
